app/controllers/users_controller.rb

```diff
@@ -88,7 +88,10 @@ def destroy
   end
 
   def bulk_destroy
-    @users = load_selected_objects(User)
+    @users = User.accessible_by(current_ability)
+    @users = apply_search_if_given(User, @users)
+    @users = load_selected_objects(@users)
```

Comment:
I wonder if we should create an additional method to DRY up these three lines. They are used in several places. Let's discuss. I'm curious what's going on in responses controller.